### ISSUE TITLE
bug with empty string args

### ISSUE THREAD

**Issue by CarloLucibello** [guildai/guildai#171]
With string argparse arguments, as in 
```python
parser.add_argument("--load-model", type=str, default="model")
```
I get a wrong behaviour when dealing with empty strings. Running the script without guild works fine 
```
python myscript.py --load-model=""
.. inside the program...
args.load_model == ""   <- correct
```
while with guild the empty string turns into the string `""`:
```
guild run myscript.py load-model=""
.. inside the program...
args.load_model == "\"\""   <- wrong
```
Same problem with `default=""` for argparse and passing no arguments.

**Comment by gar1t** [comment]
Yuck!

Stay tuned for a fix. This is a high priority bug.

I've created an issue resolution doc to help recreate and later verify the fix.

Thanks for report!

**Comment by gar1t** [comment]
This is fixed in master and will be available in the next pre-release.

**Comment by CarloLucibello** [comment]
thanks! any plan for when the next pre-release will be out?

**Comment by gar1t** [comment]
Should be fixed in 0.7.0. If you're still seeing this behavior, please feel free to re-open. Thanks again!
